go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/1dustindavis/gorilla
```

Comment:
This file is missing some indirect dependencies. If you run `go get -u` and then `go mod tidy`, it should get updated with all of the latest dependencies,

Reply:
Commit 2727219 to the rescue.